extensions/amp-a4a/rtc-documentation.md

```diff
@@ -96,14 +98,15 @@ While all three parameters of rtc-config are optional, either "vendors" or "urls
 
 ### Vendor URL Specification
 
-To spare publishers the details of having to construct URLs for external vendors, vendors may register a URL with macros in a central file called [callout-vendors.js ](https://github.com/ampproject/amphtml/blob/master/extensions/amp-a4a/0.1/callout-vendors.js), which maps unique vendor names to an object which includes a URL and a whitelist of macros. Vendors may include these macros in their URLs, which publishers can then specify the value for. For instance:
+To spare publishers the details of having to construct URLs for external vendors, vendors may register a URL with macros in a central file called [callout-vendors.js ](https://github.com/ampproject/amphtml/blob/master/extensions/amp-a4a/0.1/callout-vendors.js), which maps unique vendor names to an object which includes a URL and a whitelist of macros that can be substituted into "url". Vendors may include these macros in their URLs, which publishers can then specify the value for. Additionally, vendors may specify an "errorReportingUrl". This errorReportingUrl will be sent 1% sampled errors from callouts to their RTC endpoint. For instance:
```

Comment:
Here and above you use double quotes around property names, but below you use back-ticks (`). Might be better to stick with a consistent form.

Reply:
done